tools/trace_analyzer_test.cc

```diff
@@ -669,6 +680,144 @@ TEST_F(TraceAnalyzerTest, DeleteRange) {
   CheckFileContent(top_qps, file_path, true);
 }
 
+// Test analyzing of Iterator
+TEST_F(TraceAnalyzerTest, Iterator) {
+  std::string trace_path = test_path_ + "/trace";
+  std::string output_path = test_path_ + "/iterator";
+  std::string file_path;
+  std::vector<std::string> paras = {"./trace_analyzer",
+                                    "-analyze_get",
+                                    "-analyze_put",
+                                    "-analyze_delete",
+                                    "-analyze_merge",
+                                    "-analyze_single_delete",
+                                    "-analyze_range_delete",
+                                    "-analyze_iterator",
```

Comment:
For the iterator test you don't need to analyze other operations like get/put/delete/etc, right?

Reply:
Yeah.  I will remove the other unrelated analyzing options